build/nsswitch.conf

```diff
@@ -0,0 +1 @@
+hosts: files dns
```

Comment:
no trailing newline?

also I feel like this maybe belongs in a subdir, but I don't know where.

Reply:
first is done, don't know what subdir either...